script/conftest.py

```diff
@@ -0,0 +1,6 @@
+"""PyTest configuration."""
+
+
+def pytest_configure():
+    """Generate SQL files before running tests."""
+    exec(open("script/generate_sql").read())
```

Comment:
I may be wrong, but I don't think we need this at `script/conftest.py`, we should only need it at `tests/conftest.py`

Reply:
You are definitely right, I accidentally copy pasted it there.